dex-it/src/test/scala/com/wavesplatform/it/sync/api/GetSettingsTestSuite.scala

```diff
@@ -18,6 +18,10 @@ class GetSettingsTestSuite extends MatcherSuiteBase {
       dex1.api.settings.networkByte shouldBe 'Y'.toByte
     }
 
+    "return matcher's public key" in {
+      dex1.api.settings.matcherPublicKey should be ("7KNgQbmQj2E5sxuzDQxiXNGLmdkKKY83DHYFJ6iX93Gf")
```

Comment:
I think it could be better to use `matcher.publicKey.toString` instead of hardcode

Reply:
ok. fixed